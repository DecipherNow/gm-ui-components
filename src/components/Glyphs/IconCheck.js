import React from "react";
import Icon from "../Icon";

export default function Check(props) {
  return (
    <Icon {...props} glyphName="Check">
      <path
        d="M17.7928932,6.7928932 C18.1834175,6.4023689 18.8165825,6.4023689 19.2071068,6.7928932 C19.5976311,7.18341749 19.5976311,7.81658247 19.2071068,8.20710677 L9.99999999,17.4142136 L5.29289319,12.7071068 C4.9023689,12.3165825 4.9023689,11.6834175 5.29289319,11.2928932 C5.68341748,10.9023689 6.31658247,10.9023689 6.70710676,11.2928932 L9.99999999,14.5857864 L17.7928932,6.7928932 Z"
         
        fillRule="nonzero"
      />
    </Icon>
  );
}

Check.displayName = "Check";
