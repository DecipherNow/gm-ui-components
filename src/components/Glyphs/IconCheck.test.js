import React from "react";
import { shallow } from "enzyme";

import IconCheck from "./IconCheck.svg";

describe("IconCheck", () => {
  it("matches snapshot", () => {
    const aIconCheck = shallow(<IconCheck />).find("IconCheck.svg");
    expect(aIconCheck).toMatchSnapshot();
  });
});