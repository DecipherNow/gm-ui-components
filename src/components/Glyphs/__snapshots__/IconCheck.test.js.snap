// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconCheck matches snapshot 1`] = `<IconMock />`;
